app/api/schema/events.py

```diff
@@ -96,8 +96,8 @@ def validate_timezone(self, data, original_data):
     pentabarf_url = fields.Url(dump_only=True)
     ical_url = fields.Url(dump_only=True)
     xcal_url = fields.Url(dump_only=True)
-    live_stream_url = fields.Url(allow_none=True)
-    webinar_url = fields.Url(allow_none=True)
+    live_stream_url = fields.String(allow_none=True)
+    webinar_url = fields.String(allow_none=True)
```

Comment:
Why? These should be URLs.

If none is allowed, and none is sent from frontend, it shouldn't be an issue